### DISCUSSION TITLE
Colab environment set-up instructions - cannot find main.py

### DISCUSSION THREAD

**Discussion by amethystocean** [Comfy-Org/ComfyUI#377] [Q&A]
Hello, I'm trying to use the ComfyUI project in my own Colab environment but I'm having some trouble. I cloned the repo to my Google Drive and tried running the environment setup in a Colab notebook, but I'm getting an error related to cannot find main.py.

I was wondering if someone could provide some guidance on the necessary file structure and any additional steps needed to properly set up and use this project in my own Colab environment. Any help would be greatly appreciated. Thank you

**Comment by reese-boop** [comment]
Seems like opening the dir in gdrive shows the path. use an llm to transcribe the path correctly.

**Comment by inerdmuoiononellamerd** [comment]
grullo
